### PR TITLE
[MIG] edi_oca: Migration to 14.0

### PR DESCRIPTION
**Pull request by etobella**
This PR migrates edi to 14.0 and changes the name to edi_oca. The commit history has been changed in order to have the proper names using `git filter-branch`.

Depends on:

- [x] #370
- [x] https://github.com/OCA/connector/pull/404

@simahawk @pedrobaeza

### CODE REVIEW

File: edi_oca/security/ir_model_access.xml [hunk 1]
```diff
@@ -72,17 +72,26 @@
         <field name="perm_write" eval="1" />
         <field name="perm_unlink" eval="1" />
     </record>
+    <record model="ir.model.access" id="access_edi_exchange_record_create_user">
+        <field name="name">access_edi_exchange_record user</field>
```
Comment (simahawk) [inline edi_oca/security/ir_model_access.xml:76]: what is this? Users can alread create records w/ the rule above.

File: edi_oca/__manifest__.py [hunk 2]
```diff
@@ -8,11 +8,12 @@
...
     """,
-    "version": "13.0.1.17.0",
+    "version": "14.0.1.0.0",
+    "website": "https://github.com/OCA/edi",
     "development_status": "Alpha",
     "license": "LGPL-3",
-    "author": "ACSONE,Odoo Community Association (OCA)",
-    "maintainers": ["simahawk"],
+    "author": "ACSONE,Creu Blanca,Odoo Community Association (OCA)",
+    "maintainers": ["simahawk", "etobella"],
```
Comment (simahawk) [inline edi_oca/__manifest__.py:16]: :partying_face:

**Review by simahawk — CHANGES_REQUESTED** [review]
strange rule

**Comment by simahawk** [comment]
worth to mention in the README that the module has been renamed maybe?

Also, nothing to do for the migration?

Base PR: merged. Connector PR: asked something,

File: edi_oca/security/ir_model_access.xml (hunk 1 above)
Comment (etobella) [reply edi_oca/security/ir_model_access.xml:76]: This rule is for the wizard, on 14.0 we need to add permissions for the wizards too :wink:

Comment (simahawk) [reply edi_oca/security/ir_model_access.xml:76]: ahh sorry, I was looking only at the diff of the mig commit and I missed that the wiz name is `edi.exchange.record.create`. 
Not a blocker, but IMO it should be renamed to `edi.exchange.record.create.wiz`. What do you think?

Comment (etobella) [reply edi_oca/security/ir_model_access.xml:76]: sure!

**Comment by etobella** [comment]
@simahawk All comments attended

**Comment by simahawk** [comment]
Nothing to do for the migration?

**Comment by etobella** [comment]
Not really, it was an easy migration. We will need to add the change of name on OpenUpgrade, but it was easy :wink:

**Review by simahawk — APPROVED** [review]

**Review by albariera — APPROVED** [review]

**Pull request merged**